File: Beginner/Highest_and_Position_1080/main.go
package main

import "fmt"

func main() {
	var highest int
	numberIndex := 1
	fmt.Scan(&highest)
	var actualNumber int
	for index := 2; index <= 100; index++ {
		fmt.Scan(&actualNumber)
		if actualNumber > highest {
			highest = actualNumber
			numberIndex = index
		}
	}
	fmt.Println(highest)
	fmt.Println(numberIndex)
}

File: Beginner/Highest_and_Position_1080/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int highestNumber;
        int numberIndex = 1;
        int actualNumber;
        highestNumber = sc.nextInt();
        for (int index = 2; index <= 100; index++) {
            actualNumber = sc.nextInt();
            if (actualNumber > highestNumber) {
                highestNumber = actualNumber;
                numberIndex = index;
            }
        }
        System.out.println(highestNumber);
        System.out.println(numberIndex);
        sc.close();
    }
}